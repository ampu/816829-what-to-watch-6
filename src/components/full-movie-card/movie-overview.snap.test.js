import React from 'react';

import {render} from '@testing-library/react';

import MovieOverview from './movie-overview';

describe(`MovieOverview component`, () => {
  it(`Should render according to the snapshot with no movie provided`, () => {
    const {container} = render(<MovieOverview/>);
    expect(container).toMatchSnapshot();
  });

  it(`Should render according to the snapshot with movie provided`, () => {
    const movie = {
      id: `fakeId`,
      description: `fakeDescription`,
      director: `fakeDirector`,
      stars: [`fakeStarOne`, `fakeStarTwo`],
      rating: 9.9,
      scoresCount: 15667,
    };
    const {container} = render(<MovieOverview movie={movie}/>);
    expect(container).toMatchSnapshot();
  });
});
