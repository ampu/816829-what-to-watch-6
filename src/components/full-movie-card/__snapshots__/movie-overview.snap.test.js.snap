// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieOverview component Should render according to the snapshot with movie provided 1`] = `
<div>
  <div
    class="movie-rating"
  >
    <div
      class="movie-rating__score"
    >
      9,9
    </div>
    <p
      class="movie-rating__meta"
    >
      <span
        class="movie-rating__level"
      />
      <span
        class="movie-rating__count"
      >
        15Â 667
         ratings
      </span>
    </p>
  </div>
  <div
    class="movie-card__text"
  >
    <p>
      fakeDescription
    </p>
    <p
      class="movie-card__director"
    >
      <strong>
        Director: 
        fakeDirector
      </strong>
    </p>
    <p
      class="movie-card__starring"
    >
      <strong>
        Starring: 
        fakeStarOne, fakeStarTwo
      </strong>
    </p>
  </div>
</div>
`;

exports[`MovieOverview component Should render according to the snapshot with no movie provided 1`] = `
<div>
  <div
    class="movie-card__text"
  >
    
    
  </div>
</div>
`;
